solidity/contracts/converter/interfaces/IConverter.sol

```diff
@@ -93,16 +93,16 @@ interface IConverter is IOwned {
     /**
      * @dev triggered when a conversion between two tokens occurs
      *
-     * @param fromToken source reserve token
-     * @param toToken target reserve token
+     * @param sourceToken source reserve token
+     * @param targetToken target reserve token
      * @param trader wallet that initiated the trade
      * @param amount input amount in units of the source token
      * @param returnAmount output amount minus conversion fee in units of the target token
      * @param conversionFee conversion fee in units of the target token
      */
     event Conversion(
-        IReserveToken indexed fromToken,
-        IReserveToken indexed toToken,
+        IReserveToken indexed sourceToken,
+        IReserveToken indexed targetToken,
         address indexed trader,
         uint256 amount,
         uint256 returnAmount,
```

Comment:
Should also rename `amount` to `sourceAmount` and `returnAmount` to `targetAmount` IMO.
Another option is `inputAmount` and `outputAmount`.
It should then also be applied in function `dispatchConversionEvent` in the standard-pool converter.

Reply:
Also, in function `doConversion` in the standard-pool converter, we are still using local variables named `fromAmount` and `toAmount `, which we probably want to rename to `sourceAmount` and `targetAmount` as part of this task.

Reply:
👍